app/src/main/java/com/example/dine_and_donate/Activities/HomeActivity.java

```diff
@@ -5,7 +5,10 @@
 import android.content.BroadcastReceiver;
 import android.content.Context;
 import android.content.Intent;
+<<<<<<< HEAD
+=======
 
+>>>>>>> 12f5306b93572b8957381f37235cb51514924ab8
```

Comment:
I forgot that since my pull request was not accepted, everything that I was committing to my branch after the request was made was also present in that pull request. I resolved the merge conflicts much earlier on in order for the app to compile.
 I also look over my code for style before I create pull requests but not so much for when I am just pushing to my branch. I do this so that if I were to refactor large parts of my code and the app breaks, I can always get the old version from my branch.

Reply:
yea no worries! i saw you fixed it later. i think the issue is that when you send a pull request, every commit you make after is automatically added to the PR until it's merged.